react-app/src/navigation/AppRoutes.ts

```diff
@@ -5,6 +5,7 @@ const AppRoutes = Object.freeze({
   ProposalDetail: "/proposals/:id",
   NewProposal: "/proposals/new",
   Portfolio: "/portfolio",
+  ErrorInvalidAddress: "/error/invalid-address",
```

Comment:
i think no need the `/error` prefix, as `invalid-address` implies there's an error

Reply:
updated 🙇